pkgs/altera-quartus/generic.nix

```diff
@@ -425,7 +425,6 @@ stdenv.mkDerivation rec {
             maybe_proot_cmd="proot -b ${bash}/bin/sh:/bin/sh"
             # Work around bug with linux 4.8.4+ (costs some performance, but
             # prevents breakage).
-            export PROOT_NO_SECCOMP=1
```

Comment:
The comment above should be removed as well.

Reply:
FYI, I currently depend on nixpkgs-17.03 for this expression at $DAYJOB. I'm in the process of updating to latest stable, but until that proot fix lands in a stable nixpkgs branch (and I've migrated to it), I cannot integrate this.

Would you mind if I pick other parts of this PR / do some refactorings?